src/Controller/AdminController.php

```diff
@@ -139,6 +156,74 @@ public function check(RequestDataBag $dataBag): JsonResponse
         }
     }
 
+    /**
+     * Send a capture request to the Adyen platform
+     *
+     * @Route(
+     *     "/api/adyen/capture",
```

Comment:
Are these routes accessible only to requests coming from the admin?

Reply:
@acampos1916 yes they are, shopware manages that with the `@RouteScope(scopes={"administration"})` at the top of the controller class